src/cpu_data.go

```diff
@@ -141,52 +143,63 @@ func NewCollector(handle func() (collect.AMDParams)) prometheus.Collector {
 		GPUDevId: prometheus.NewDesc(
 			prometheus.BuildFQName("amd", "", "gpu_dev_id"),
 			"AMD Params",// The metric's help text.
-			[]string{"gpu_dev_id", "productname"},// The metric's variable label dimensions.
+			[]string{"gpu_dev_id", "productname", "node_id", "pod"},// The metric's variable label dimensions.
 			nil,// The metric's constant label dimensions.
 		),
 		GPUPowerCap: prometheus.NewDesc(
 			prometheus.BuildFQName("amd", "", "gpu_power_cap"),
 			"AMD Params",// The metric's help text.
-			[]string{"gpu_power_cap", "productname"},// The metric's variable label dimensions.
+			[]string{"gpu_power_cap", "productname", "node_id", "pod"},// The metric's variable label dimensions.
 			nil,// The metric's constant label dimensions.
 		),
 		GPUPowerAvg: prometheus.NewDesc(
 			prometheus.BuildFQName("amd", "", "gpu_power_avg"),
 			"AMD Params",// The metric's help text.
-			[]string{"gpu_power_avg", "productname"},// The metric's variable label dimensions.
+			[]string{"gpu_power_avg", "productname", "node_id", "pod"},// The metric's variable label dimensions.
 			nil,// The metric's constant label dimensions.
 		),
 		GPUTemperature: prometheus.NewDesc(
 			prometheus.BuildFQName("amd", "", "gpu_current_temperature"),
 			"AMD Params",// The metric's help text.
-			[]string{"gpu_current_temperature", "productname"},// The metric's variable label dimensions.
+			[]string{"gpu_current_temperature", "productname", "node_id", "pod"},// The metric's variable label dimensions.
 			nil,// The metric's constant label dimensions.
 		),
 		GPUSCLK: prometheus.NewDesc(
 			prometheus.BuildFQName("amd", "", "gpu_SCLK"),
 			"AMD Params",// The metric's help text.
-			[]string{"gpu_SCLK", "productname"},// The metric's variable label dimensions.
+			[]string{"gpu_SCLK", "productname", "node_id", "pod"},// The metric's variable label dimensions.
 			nil,// The metric's constant label dimensions.
 		),
 		GPUMCLK: prometheus.NewDesc(
 			prometheus.BuildFQName("amd", "", "gpu_MCLK"),
 			"AMD Params",// The metric's help text.
-			[]string{"gpu_MCLK", "productname"},// The metric's variable label dimensions.
+			[]string{"gpu_MCLK", "productname", "node_id", "pod"},// The metric's variable label dimensions.
 			nil,// The metric's constant label dimensions.
 		),
 		GPUUsage: prometheus.NewDesc(
                         prometheus.BuildFQName("amd", "", "gpu_use_percent"),
                         "AMD Params",// The metric's help text.
-                        []string{"gpu_use_percent", "productname"},// The metric's variable label dimensions.
+                        []string{"gpu_id", "productname", "node_id", "pod"},// The metric's variable label dimensions.
                         nil,// The metric's constant label dimensions.
                 ),
-                GPUMemoryUsage: prometheus.NewDesc(
+                GPUMemoryBusyPercent: prometheus.NewDesc(
```

Comment:
By "busy" memory, does it mean "used"? Not sure what 'busy' means for memory? Could we changed to "used"?